docs/guide/src/dev/protobuf.md

```diff
@@ -33,7 +33,7 @@ unzip protoc-24.4-linux-x86_64.zip -d ~/.local/
 ## Installing buf
 
 The `buf` tool is required to update lockfiles used for version management in
-the [Buf Schema Registry](https://buf.build.penumbra-zone/penumbra). Visit
+the [Buf Schema Registry](https://buf.build/penumbra-zone/penumbra/docs/main:penumbra.core.transaction.v1alpha1). Visit
```

Comment:
Can we leave this as https://buf.build/penumbra-zone/penumbra so that it shows the list of all packages?

Reply:
done